src/site/resources/js/checkstyle.js

```diff
@@ -0,0 +1,27 @@
+/*jslint browser: true */
+/*jslint es6: true */
+/*jshint esversion: 6 */
+/*global window */
+let scrollButton;
+const scrollDistanceToButtonVisibility = 500;
+
+window.addEventListener("load", function () {
+    "use strict";
```

Comment:
I thought this can be added to top of the script instead of adding it to every function

Reply:
it does not work at most top for some reason.